### DISCUSSION TITLE
Fermata auto liste mac IPTV

### DISCUSSION THREAD

**Discussion by carlo201983** [AndreyPavlenko/Fermata#288] [Ideas]
idea, would there be the possibility of inserting the function to insert Mac lists. ? obviously referring to IPTV

**Comment by AndreyPavlenko** [comment]
What is "mac IPTV"?

**Reply by carlo201983** [reply]
I was wondering if you could include the possibility of also inserting other list formats. that's all. Thanks

**Reply by AndreyPavlenko** [reply]
Currently, only m3u playlists are supported. Support for other formats will require some development efforts and time.

**Comment by carlo201983** [comment]
thanks for replying, iptv lists are in 3 formats: m3u - user/pass - mac.
for the MAC lists you enter the server and the Mac.. I'll attach a video of iptvextreme so you can understand better..

________________________________
Da: Andrey Pavlenko ***@***.***>
Inviato: venerdì 8 settembre 2023 12:08
A: AndreyPavlenko/Fermata ***@***.***>
Cc: carlo201983 ***@***.***>; Author ***@***.***>
Oggetto: Re: [AndreyPavlenko/Fermata] Fermata auto liste mac IPTV (Discussion #288)


What is "mac IPTV".

**Reply by carlo201983** [reply]
Hello. Will you consider my request? You didn't answer me anymore.

**Reply by AndreyPavlenko** [reply]
Hello. You said that you will attach a video for clarification. Currently, I do not understand the request.

**Comment by carlo201983** [comment]
https://github.com/AndreyPavlenko/Fermata/assets/144391606/385fe307-5e67-4db9-8d18-c4eb168ce8c1

I replied to you via email, I thought you would have seen the attachment..

**Reply by AndreyPavlenko** [reply]
No, I haven't.

**Comment by carlo201983** [comment]
I wish I had been more explicit. IPTV lists are of 3 types: m3u, user pass, mac
https://github.com/AndreyPavlenko/Fermata/assets/144391606/3cb475c5-8bbf-4e87-ab10-90a82ba43945

**Reply by AndreyPavlenko** [reply]
I still do not understand what is this. Is it some proprietary protocol? Is there any documentation? What the mac address is used for?

**Reply by carlo201983** [reply]
oh well, never mind... I give up, I don't know how else to explain it. thanks anyway.. I'll see if I can find some other software, or some other way to see. HI

**Reply by cazuza82** [reply]
Carlo201983 you find away to see iptv for mac in Fermata auto ?? I try the same but not luck yet

**Reply by carlo201983** [reply]
Hi, no, unfortunately you can't add Mac lists. only he knows if the author wants to implement it
